README.md

```diff
@@ -1 +1,34 @@
-# Portfolio-setup-and-mobile-version-skeleton
\ No newline at end of file
+## Portfolio-setup-and-mobile-version-skeleton
+
```

Comment:
- [ ] This section should have the project title as a heading and a description as a paragraph.